src/main.c

```diff
@@ -118,6 +121,54 @@ static int process_imu(const struct device *dev) {
     return 0;
 }
 
+static int process_baro(const struct device *dev) {
+    struct sensor_value temp;
+    struct sensor_value press;
+
+    float temp_f;
```

Comment:
What does the `_f` suffix stand for?

Reply:
Stands for the float value, used to distinguish from the `temp` and `press`

Reply:
With modern code editors or IDEs, finding a type of a variable is a simple hover over it, I don't think we need this personally.

Reply:
I agree, thanks!